### PR TITLE
license-classifications: Improve what gets assigned to the 'include-in-notice-file' category

### PR DESCRIPTION
**Pull request by fviernau**
See individual commits.

Context: #66.

### CODE REVIEW

File: tools/curations/buildSrc/src/main/kotlin/ScanCodeLicenseDbClassifications.kt [hunk 1]
```diff
@@ -57,6 +57,10 @@ private val CONTRIBUTOR_LICENSE_AGREEMENT_IDS = listOf(
     "LicenseRef-scancode-square-cla"
 ).map { SpdxSingleLicenseExpression.parse(it) }.toSet()
 
+private const val CATEGORY_CONTRIBUTOR_LICENSE_AGREEMENT = "contributor-license-agreement"
+private const val CATEGORY_GENERIC = "generic"
+private const val CATEGORY_UNKNOWN = "unknown"
```
Comment (sschuberth) [inline tools/curations/buildSrc/src/main/kotlin/ScanCodeLicenseDbClassifications.kt:62]: Nit: The extraction of these constants is unrelated to the commit topic.

**Review by sschuberth — APPROVED** [review]

**Comment by pombredanne** [comment]
@fviernau Could these categories be included in the ScanCode license DB directly to avoid data fragmentation? They are likely overlapping with some of the license flags .... I guess you are on top of it :)
- See https://github.com/nexB/scancode-toolkit/issues/3038
- See https://github.com/nexB/scancode-toolkit/issues/3147 (yours)
